sdk/src/utils/font_thumbnail.rs

```diff
@@ -291,16 +288,36 @@ fn measure_text_in_buffer(buffer: &mut BorrowedWithFontSystem<Buffer>) -> Result
 
 impl From<Arc<Font>> for FontNameInfo {
     fn from(font: Arc<Font>) -> Self {
+        // The US English language ID; currently localization is still a work in
+        // progress
+        const US_EN_LANGUAGE_ID: u16 = 0x0409;
+
         let face = font.rustybuzz();
-        let mut full_name = None;
-        let mut sample_text = None;
-        for name in face.names() {
-            match name.name_id {
-                FULL_NAME_ID => full_name = name.to_string(),
-                SAMPLE_TEXT_ID => sample_text = name.to_string(),
-                _ => {}
-            }
-        }
+        // We want to use PlatformID::Unicode/LanguageID::English for the name table when possible,
+        // if not available, we will look for Windows, and then finally Macintosh
+        let preferred_search_order = [
+            (PlatformId::Unicode, US_EN_LANGUAGE_ID, 3), // Unicode, English, Unicode BMP encoding
```

Comment:
Might be better to have the encoding IDs be defined with constants?  (would remove the need for the comment)